tools/ci_build/github/azure-pipelines/mac-ios-packaging-pipeline.yml

```diff
@@ -119,31 +119,3 @@ stages:
       skipComponentGovernanceDetection: true
 
     timeoutInMinutes: 10
```

Comment:
let's just remove the whole IosPackaging_TestPackageSwift_Full stage

Reply:
right, done.